core/form.go

```diff
@@ -190,7 +194,7 @@ func (fm *Form) Init() {
 					}
```

Comment:
You are still not recomputing doc, readOnlyTag, def, hasDef, etc every time. This does **not** address #1098; that really is for rare cases, which are different from those this PR addresses.